specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2025-06-01/workspaceRP.json

```diff
@@ -1924,6 +1924,9 @@
```

Comment:
>"type": "boolean" [](http://example.com/codeflow?start=10&length=17)

Consider enum that gives a clear name to each state vs. boolean (true/false only). This could also be more extensible, e.g. encryptionType: { None, ServerCMK, ... }